src/control/lib/control/pool.go

```diff
@@ -1241,6 +1240,7 @@ func GetMaxPoolSize(ctx context.Context, log logging.Logger, rpcClient UnaryInvo
 	}
 
 	hostStorageMap := resp.HostStorage
+	ranksWithoutSmd := make(map[system.Rank]*storage.ScmNamespace, 0)
```

Comment:
nit: It's unusual to provide a hint to `make()` for map initialization. It's more of a performance optimization for situations where the eventual (large) number of keypairs is known. It's not wrong, but it did make me do a double-take because I've only ever seen hints used for preallocation of slices.

Reply:
- [ ] Remove useless make capacity argument